.circleci/config.yml

```diff
@@ -951,6 +952,46 @@ jobs:
           artifact_name: LTE AGW
           version_path: versions/magma_version
 
+  lte-agw-aws:
+    environment:
+      #AWS_ACCESS_KEY_ID: "$AWS_FB_CRED"
+      #PARAM_AWS_SECRET_ACCESS_KEY: "$(printenv AWS_FB_SECRET)"
+      MAGMA_ROOT: /home/circleci/project  
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - python/set_version
+      - run:
+          command: |
+            pip install wheel
```

Comment:
can you do all pip install at once? Can we use pip3?

Reply:
done